2024-12-03 17:59:25,668 - INFO - Initializing Speech-to-Speech Bot...
2024-12-03 17:59:50,614 - INFO - Initializing Speech-to-Speech Bot...
2024-12-03 18:00:26,642 - INFO - Initializing Speech-to-Speech Bot...
2024-12-03 18:02:50,255 - INFO - Initializing Speech-to-Speech Bot...
2024-12-03 18:03:20,726 - INFO - Initializing Speech-to-Speech Bot...
2024-12-03 18:11:34,046 - INFO - Initializing Speech-to-Speech Bot...
2024-12-03 18:13:28,550 - INFO - Initializing Speech-to-Speech Bot...
2024-12-03 18:13:46,814 - INFO - Initializing Speech-to-Speech Bot...
2024-12-03 18:14:23,519 - INFO - Initializing Speech-to-Speech Bot...
2024-12-03 18:14:50,252 - INFO - Initializing Speech-to-Speech Bot...
2024-12-03 18:14:57,263 - ERROR - Speech Recognition could not understand the audio.
2024-12-03 18:14:57,283 - INFO - Initializing Speech-to-Speech Bot...
2024-12-03 18:15:31,323 - INFO - Initializing Speech-to-Speech Bot...
2024-12-03 18:15:48,044 - INFO - Initializing Speech-to-Speech Bot...
2024-12-03 18:18:05,313 - INFO - Initializing Speech-to-Speech Bot...
2024-12-03 18:18:10,544 - ERROR - An error occurred: listening timed out while waiting for phrase to start
2024-12-03 18:18:29,587 - INFO - Initializing Speech-to-Speech Bot...
2024-12-03 18:18:34,750 - ERROR - An error occurred: listening timed out while waiting for phrase to start
2024-12-03 18:18:40,387 - INFO - Initializing Speech-to-Speech Bot...
2024-12-03 18:18:45,540 - ERROR - An error occurred: listening timed out while waiting for phrase to start
2024-12-03 18:19:50,539 - INFO - Initializing Speech-to-Speech Bot...
2024-12-03 18:19:55,759 - ERROR - An error occurred: listening timed out while waiting for phrase to start
2024-12-03 18:22:16,476 - INFO - Initializing Speech-to-Speech Bot...
2024-12-03 18:22:27,653 - ERROR - Error during speech recognition: listening timed out while waiting for phrase to start
2024-12-03 18:22:27,666 - ERROR - No input detected during speech recognition.
2024-12-03 18:22:27,667 - INFO - Initializing Speech-to-Speech Bot...
2024-12-03 18:22:38,790 - ERROR - Error during speech recognition: listening timed out while waiting for phrase to start
2024-12-03 18:22:38,806 - ERROR - No input detected during speech recognition.
2024-12-03 18:22:38,809 - INFO - Initializing Speech-to-Speech Bot...
2024-12-03 18:22:49,936 - ERROR - Error during speech recognition: listening timed out while waiting for phrase to start
2024-12-03 18:22:49,949 - ERROR - No input detected during speech recognition.
2024-12-03 18:24:20,683 - INFO - Initializing Speech-to-Speech Bot...
2024-12-03 18:24:31,872 - ERROR - Error during speech recognition: listening timed out while waiting for phrase to start
2024-12-03 18:24:31,887 - ERROR - No input detected during speech recognition.
2024-12-03 18:24:48,257 - INFO - Initializing Speech-to-Speech Bot...
2024-12-03 18:24:59,411 - ERROR - Error during speech recognition: listening timed out while waiting for phrase to start
2024-12-03 18:24:59,429 - ERROR - No input detected during speech recognition.
2024-12-03 18:27:03,953 - INFO - Initializing Speech-to-Speech Bot...
2024-12-03 18:27:08,103 - ERROR - Speech Recognition could not understand the audio.
2024-12-03 18:27:08,108 - ERROR - No input detected during speech recognition.
2024-12-03 18:28:29,381 - INFO - Initializing Speech-to-Speech Bot...
2024-12-03 18:28:45,558 - ERROR - Error during speech recognition: listening timed out while waiting for phrase to start
2024-12-03 18:28:45,575 - ERROR - No input detected during speech recognition.
2024-12-03 18:28:45,576 - INFO - Initializing Speech-to-Speech Bot...
2024-12-03 18:29:01,717 - ERROR - Error during speech recognition: listening timed out while waiting for phrase to start
2024-12-03 18:29:01,738 - ERROR - No input detected during speech recognition.
2024-12-03 18:29:01,739 - INFO - Initializing Speech-to-Speech Bot...
2024-12-03 18:29:17,871 - ERROR - Error during speech recognition: listening timed out while waiting for phrase to start
2024-12-03 18:29:17,889 - ERROR - No input detected during speech recognition.
2024-12-03 18:31:23,325 - INFO - Initializing Speech-to-Speech Bot...
2024-12-03 18:31:39,522 - ERROR - No input detected during speech recognition.
2024-12-03 18:31:57,166 - INFO - Initializing Speech-to-Speech Bot...
2024-12-03 18:32:13,304 - ERROR - No input detected during speech recognition.
2024-12-03 18:32:31,251 - INFO - Initializing Speech-to-Speech Bot...
2024-12-03 18:32:47,472 - ERROR - No input detected during speech recognition.
2024-12-03 18:34:36,311 - INFO - Initializing Speech-to-Speech Bot...
2024-12-03 18:35:07,530 - ERROR - No input detected during speech recognition.
2024-12-03 19:51:51,017 - INFO - Initializing Speech-to-Speech Bot...
2024-12-03 19:52:22,636 - ERROR - No input detected during speech recognition.
2024-12-03 19:52:22,640 - INFO - Initializing Speech-to-Speech Bot...
2024-12-03 19:52:53,803 - ERROR - No input detected during speech recognition.
2024-12-03 21:07:19,163 - INFO - Initializing Speech-to-Speech Bot...
2024-12-03 21:07:50,947 - ERROR - No input detected during speech recognition.
2024-12-03 22:17:09,800 - INFO - Initializing Speech-to-Speech Bot...
2024-12-03 22:17:41,459 - ERROR - No input detected during speech recognition.
2024-12-05 21:16:06,470 - INFO - Initializing Speech-to-Speech Bot...
2024-12-05 21:16:18,274 - INFO - Could not import comtypes.gen, trying to create it.
2024-12-05 21:16:18,274 - INFO - Created comtypes.gen directory: 'C:\Users\lodhi\AppData\Local\Programs\Python\Python312\Lib\site-packages\comtypes\gen'
2024-12-05 21:16:18,274 - INFO - Writing __init__.py file: 'C:\Users\lodhi\AppData\Local\Programs\Python\Python312\Lib\site-packages\comtypes\gen\__init__.py'
2024-12-05 21:16:18,289 - INFO - Using writeable comtypes cache directory: 'C:\Users\lodhi\AppData\Local\Programs\Python\Python312\Lib\site-packages\comtypes\gen'
2024-12-05 21:16:18,304 - INFO - Could not import comtypes.gen._C866CA3A_32F7_11D2_9602_00C04F8EE628_0_5_4: No module named 'comtypes.gen._C866CA3A_32F7_11D2_9602_00C04F8EE628_0_5_4'
2024-12-05 21:16:18,308 - INFO - # Generating comtypes.gen._C866CA3A_32F7_11D2_9602_00C04F8EE628_0_5_4
2024-12-05 21:16:18,376 - INFO - # Generating comtypes.gen.SpeechLib
2024-12-05 21:16:18,385 - INFO - Could not import comtypes.gen._00020430_0000_0000_C000_000000000046_0_2_0: No module named 'comtypes.gen._00020430_0000_0000_C000_000000000046_0_2_0'
2024-12-05 21:16:18,386 - INFO - # Generating comtypes.gen._00020430_0000_0000_C000_000000000046_0_2_0
2024-12-05 21:16:18,390 - INFO - # Generating comtypes.gen.stdole
2024-12-05 21:16:59,182 - INFO - Initializing Speech-to-Speech Bot...
2024-12-05 21:17:18,148 - ERROR - Speech Recognition could not understand the audio.
2024-12-05 21:17:18,213 - ERROR - No input detected during speech recognition.
2024-12-05 21:17:18,220 - INFO - Initializing Speech-to-Speech Bot...
2024-12-05 21:17:36,370 - INFO - Imported existing <module 'comtypes.gen' from 'C:\\Users\\lodhi\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\comtypes\\gen\\__init__.py'>
2024-12-05 21:17:36,370 - INFO - Using writeable comtypes cache directory: 'C:\Users\lodhi\AppData\Local\Programs\Python\Python312\Lib\site-packages\comtypes\gen'
2024-12-05 21:18:00,287 - INFO - Initializing Speech-to-Speech Bot...
2024-12-05 21:18:44,833 - INFO - Initializing Speech-to-Speech Bot...
2024-12-05 21:19:06,393 - ERROR - Speech Recognition could not understand the audio.
2024-12-05 21:19:06,425 - ERROR - No input detected during speech recognition.
2024-12-05 21:19:06,427 - INFO - Initializing Speech-to-Speech Bot...
2024-12-05 21:19:13,654 - ERROR - Speech Recognition could not understand the audio.
2024-12-05 21:19:13,663 - ERROR - No input detected during speech recognition.
2024-12-05 21:19:13,664 - INFO - Initializing Speech-to-Speech Bot...
2024-12-05 21:19:39,557 - INFO - Initializing Speech-to-Speech Bot...
2024-12-05 21:19:51,045 - INFO - Imported existing <module 'comtypes.gen' from 'C:\\Users\\lodhi\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\comtypes\\gen\\__init__.py'>
2024-12-05 21:19:51,045 - INFO - Using writeable comtypes cache directory: 'C:\Users\lodhi\AppData\Local\Programs\Python\Python312\Lib\site-packages\comtypes\gen'
2024-12-05 21:20:20,212 - INFO - Initializing Speech-to-Speech Bot...
2024-12-05 21:20:28,586 - ERROR - Speech Recognition could not understand the audio.
2024-12-05 21:20:28,609 - ERROR - No input detected during speech recognition.
2024-12-05 21:20:28,610 - INFO - Initializing Speech-to-Speech Bot...
2024-12-05 21:21:01,457 - INFO - Initializing Speech-to-Speech Bot...
2024-12-05 21:21:33,787 - INFO - Initializing Speech-to-Speech Bot...
2024-12-05 21:22:10,132 - INFO - Initializing Speech-to-Speech Bot...
2024-12-05 21:29:14,333 - INFO - Initializing Speech-to-Speech Bot...
2024-12-05 21:29:45,616 - ERROR - No input detected during speech recognition.
2024-12-05 21:29:45,625 - INFO - Initializing Speech-to-Speech Bot...
2024-12-05 21:30:16,783 - INFO - Imported existing <module 'comtypes.gen' from 'C:\\Users\\lodhi\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\comtypes\\gen\\__init__.py'>
2024-12-05 21:30:16,783 - INFO - Using writeable comtypes cache directory: 'C:\Users\lodhi\AppData\Local\Programs\Python\Python312\Lib\site-packages\comtypes\gen'
2024-12-05 21:30:30,073 - INFO - Initializing Speech-to-Speech Bot...
2024-12-05 21:30:42,903 - INFO - Initializing Speech-to-Speech Bot...
2024-12-05 21:31:13,994 - ERROR - No input detected during speech recognition.
2024-12-05 21:31:13,994 - INFO - Initializing Speech-to-Speech Bot...
2024-12-05 21:31:26,742 - ERROR - Speech Recognition could not understand the audio.
2024-12-05 21:31:26,755 - ERROR - No input detected during speech recognition.
2024-12-05 21:31:31,698 - INFO - Initializing Speech-to-Speech Bot...
2024-12-05 21:31:47,630 - ERROR - Speech Recognition could not understand the audio.
2024-12-05 21:31:47,643 - ERROR - No input detected during speech recognition.
2024-12-05 21:31:52,251 - INFO - Initializing Speech-to-Speech Bot...
2024-12-05 21:32:23,416 - ERROR - No input detected during speech recognition.
2024-12-05 21:32:26,400 - INFO - Initializing Speech-to-Speech Bot...
2024-12-05 21:32:32,376 - ERROR - Speech Recognition could not understand the audio.
2024-12-05 21:32:32,408 - ERROR - No input detected during speech recognition.
2024-12-05 21:32:52,556 - INFO - Initializing Speech-to-Speech Bot...
2024-12-05 21:33:10,370 - ERROR - Speech Recognition could not understand the audio.
2024-12-05 21:33:10,400 - ERROR - No input detected during speech recognition.
2024-12-05 21:34:09,913 - INFO - Initializing Speech-to-Speech Bot...
2024-12-05 21:34:41,124 - ERROR - No input detected during speech recognition.
2024-12-05 21:34:44,389 - INFO - Initializing Speech-to-Speech Bot...
2024-12-05 21:35:11,225 - INFO - Imported existing <module 'comtypes.gen' from 'C:\\Users\\lodhi\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\comtypes\\gen\\__init__.py'>
2024-12-05 21:35:11,227 - INFO - Using writeable comtypes cache directory: 'C:\Users\lodhi\AppData\Local\Programs\Python\Python312\Lib\site-packages\comtypes\gen'
2024-12-05 21:35:15,050 - INFO - Initializing Speech-to-Speech Bot...
2024-12-05 21:35:48,332 - ERROR - Speech Recognition could not understand the audio.
2024-12-05 21:35:48,363 - ERROR - No input detected during speech recognition.
2024-12-05 21:35:48,404 - INFO - Initializing Speech-to-Speech Bot...
2024-12-05 21:35:58,005 - ERROR - Speech Recognition could not understand the audio.
2024-12-05 21:35:58,038 - ERROR - No input detected during speech recognition.
2024-12-05 21:36:03,954 - INFO - Initializing Speech-to-Speech Bot...
2024-12-05 21:36:35,119 - ERROR - No input detected during speech recognition.
2024-12-05 21:36:35,125 - INFO - Initializing Speech-to-Speech Bot...
2024-12-05 21:36:45,831 - INFO - Initializing Speech-to-Speech Bot...
2024-12-05 21:37:46,807 - INFO - Initializing Speech-to-Speech Bot...
2024-12-05 21:37:54,919 - INFO - Imported existing <module 'comtypes.gen' from 'C:\\Users\\lodhi\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\comtypes\\gen\\__init__.py'>
2024-12-05 21:37:54,920 - INFO - Using writeable comtypes cache directory: 'C:\Users\lodhi\AppData\Local\Programs\Python\Python312\Lib\site-packages\comtypes\gen'
2024-12-05 21:38:18,010 - INFO - Initializing Speech-to-Speech Bot...
2024-12-05 21:39:13,598 - INFO - Initializing Speech-to-Speech Bot...
2024-12-05 21:39:26,483 - ERROR - Speech Recognition could not understand the audio.
2024-12-05 21:39:26,514 - ERROR - No input detected during speech recognition.
2024-12-05 21:39:29,534 - INFO - Initializing Speech-to-Speech Bot...
2024-12-05 21:39:42,056 - ERROR - Speech Recognition could not understand the audio.
2024-12-05 21:39:42,088 - ERROR - No input detected during speech recognition.
2024-12-05 21:39:43,857 - INFO - Initializing Speech-to-Speech Bot...
2024-12-05 21:39:51,781 - INFO - Imported existing <module 'comtypes.gen' from 'C:\\Users\\lodhi\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\comtypes\\gen\\__init__.py'>
2024-12-05 21:39:51,781 - INFO - Using writeable comtypes cache directory: 'C:\Users\lodhi\AppData\Local\Programs\Python\Python312\Lib\site-packages\comtypes\gen'
2024-12-05 21:44:09,516 - WARNING - Error while downloading from https://cdn-lfs.hf.co/microsoft/DialoGPT-small/ea50e39b0e9368b9110cc5ab48012ddbc7bd90f2b17410aed643a7404acac567?response-content-disposition=inline%3B+filename*%3DUTF-8%27%27model.safetensors%3B+filename%3D%22model.safetensors%22%3B&Expires=1733674389&Policy=eyJTdGF0ZW1lbnQiOlt7IkNvbmRpdGlvbiI6eyJEYXRlTGVzc1RoYW4iOnsiQVdTOkVwb2NoVGltZSI6MTczMzY3NDM4OX19LCJSZXNvdXJjZSI6Imh0dHBzOi8vY2RuLWxmcy5oZi5jby9taWNyb3NvZnQvRGlhbG9HUFQtc21hbGwvZWE1MGUzOWIwZTkzNjhiOTExMGNjNWFiNDgwMTJkZGJjN2JkOTBmMmIxNzQxMGFlZDY0M2E3NDA0YWNhYzU2Nz9yZXNwb25zZS1jb250ZW50LWRpc3Bvc2l0aW9uPSoifV19&Signature=Hj%7ELBGxM2B-Sge0LeuWxSSrugEG7bGK8ocoG-%7EZ3QC8efi1bJ9Lsh7Cy2vZyDcRF5%7EId%7Es-rpFeRFnmj9UU04tAhhBf2v4ZNe%7E9GDCsrcX4XlJhMX0eDsNAHlAcrSVCPAiCk76fQESqt-V6IvWGa41rf09BpVSBaHDvUaVFD-yaTSluYulzn1vmj75061LHW-%7ED17CO5fF9wVz4ImBFEKF-8AmYjWojqBZHa0stpF7UhbVOaoni1s-VTw11wBcEdUD7eedeL7jyROy%7ERDAcAU3%7EF7Em56YbIn8CWT0U1AKRVrCHNPz-BvXjqmsqFqNpWjUV8nqNnMtd68TnDnGrQEQ__&Key-Pair-Id=K3RPWS32NSSJCE: The operation did not complete (read) (_ssl.c:2559)
Trying to resume download...
2024-12-05 21:44:45,697 - WARNING - Error while downloading from https://cdn-lfs.hf.co/microsoft/DialoGPT-small/ea50e39b0e9368b9110cc5ab48012ddbc7bd90f2b17410aed643a7404acac567?response-content-disposition=inline%3B+filename*%3DUTF-8%27%27model.safetensors%3B+filename%3D%22model.safetensors%22%3B&Expires=1733674389&Policy=eyJTdGF0ZW1lbnQiOlt7IkNvbmRpdGlvbiI6eyJEYXRlTGVzc1RoYW4iOnsiQVdTOkVwb2NoVGltZSI6MTczMzY3NDM4OX19LCJSZXNvdXJjZSI6Imh0dHBzOi8vY2RuLWxmcy5oZi5jby9taWNyb3NvZnQvRGlhbG9HUFQtc21hbGwvZWE1MGUzOWIwZTkzNjhiOTExMGNjNWFiNDgwMTJkZGJjN2JkOTBmMmIxNzQxMGFlZDY0M2E3NDA0YWNhYzU2Nz9yZXNwb25zZS1jb250ZW50LWRpc3Bvc2l0aW9uPSoifV19&Signature=Hj%7ELBGxM2B-Sge0LeuWxSSrugEG7bGK8ocoG-%7EZ3QC8efi1bJ9Lsh7Cy2vZyDcRF5%7EId%7Es-rpFeRFnmj9UU04tAhhBf2v4ZNe%7E9GDCsrcX4XlJhMX0eDsNAHlAcrSVCPAiCk76fQESqt-V6IvWGa41rf09BpVSBaHDvUaVFD-yaTSluYulzn1vmj75061LHW-%7ED17CO5fF9wVz4ImBFEKF-8AmYjWojqBZHa0stpF7UhbVOaoni1s-VTw11wBcEdUD7eedeL7jyROy%7ERDAcAU3%7EF7Em56YbIn8CWT0U1AKRVrCHNPz-BvXjqmsqFqNpWjUV8nqNnMtd68TnDnGrQEQ__&Key-Pair-Id=K3RPWS32NSSJCE: HTTPSConnectionPool(host='cdn-lfs.hf.co', port=443): Read timed out.
Trying to resume download...
2024-12-05 21:46:58,649 - WARNING - Error while downloading from https://cdn-lfs.hf.co/microsoft/DialoGPT-small/ea50e39b0e9368b9110cc5ab48012ddbc7bd90f2b17410aed643a7404acac567?response-content-disposition=inline%3B+filename*%3DUTF-8%27%27model.safetensors%3B+filename%3D%22model.safetensors%22%3B&Expires=1733674389&Policy=eyJTdGF0ZW1lbnQiOlt7IkNvbmRpdGlvbiI6eyJEYXRlTGVzc1RoYW4iOnsiQVdTOkVwb2NoVGltZSI6MTczMzY3NDM4OX19LCJSZXNvdXJjZSI6Imh0dHBzOi8vY2RuLWxmcy5oZi5jby9taWNyb3NvZnQvRGlhbG9HUFQtc21hbGwvZWE1MGUzOWIwZTkzNjhiOTExMGNjNWFiNDgwMTJkZGJjN2JkOTBmMmIxNzQxMGFlZDY0M2E3NDA0YWNhYzU2Nz9yZXNwb25zZS1jb250ZW50LWRpc3Bvc2l0aW9uPSoifV19&Signature=Hj%7ELBGxM2B-Sge0LeuWxSSrugEG7bGK8ocoG-%7EZ3QC8efi1bJ9Lsh7Cy2vZyDcRF5%7EId%7Es-rpFeRFnmj9UU04tAhhBf2v4ZNe%7E9GDCsrcX4XlJhMX0eDsNAHlAcrSVCPAiCk76fQESqt-V6IvWGa41rf09BpVSBaHDvUaVFD-yaTSluYulzn1vmj75061LHW-%7ED17CO5fF9wVz4ImBFEKF-8AmYjWojqBZHa0stpF7UhbVOaoni1s-VTw11wBcEdUD7eedeL7jyROy%7ERDAcAU3%7EF7Em56YbIn8CWT0U1AKRVrCHNPz-BvXjqmsqFqNpWjUV8nqNnMtd68TnDnGrQEQ__&Key-Pair-Id=K3RPWS32NSSJCE: HTTPSConnectionPool(host='cdn-lfs.hf.co', port=443): Read timed out.
Trying to resume download...
2024-12-05 21:48:13,460 - INFO - Initializing Speech-to-Speech Bot...
2024-12-05 21:48:25,279 - INFO - Imported existing <module 'comtypes.gen' from 'C:\\Users\\lodhi\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\comtypes\\gen\\__init__.py'>
2024-12-05 21:48:25,279 - INFO - Using writeable comtypes cache directory: 'C:\Users\lodhi\AppData\Local\Programs\Python\Python312\Lib\site-packages\comtypes\gen'
2024-12-05 21:49:02,201 - INFO - Initializing Speech-to-Speech Bot...
2024-12-05 21:49:29,697 - INFO - Initializing Speech-to-Speech Bot...
2024-12-05 21:50:05,153 - INFO - Initializing Speech-to-Speech Bot...
2024-12-05 21:50:28,297 - INFO - Initializing Speech-to-Speech Bot...
2024-12-05 21:50:48,529 - INFO - Initializing Speech-to-Speech Bot...
2024-12-05 21:51:11,417 - INFO - Initializing Speech-to-Speech Bot...
2024-12-05 21:51:38,184 - INFO - Initializing Speech-to-Speech Bot...
2024-12-05 21:51:46,019 - ERROR - Speech Recognition could not understand the audio.
2024-12-05 21:51:46,064 - ERROR - No input detected during speech recognition.
2024-12-05 21:52:38,158 - INFO - Initializing Speech-to-Speech Bot...
2024-12-05 21:52:56,170 - INFO - Imported existing <module 'comtypes.gen' from 'C:\\Users\\lodhi\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\comtypes\\gen\\__init__.py'>
2024-12-05 21:52:56,170 - INFO - Using writeable comtypes cache directory: 'C:\Users\lodhi\AppData\Local\Programs\Python\Python312\Lib\site-packages\comtypes\gen'
2024-12-05 21:53:05,675 - INFO - Initializing Speech-to-Speech Bot...
2024-12-05 21:53:13,267 - ERROR - Speech Recognition could not understand the audio.
2024-12-05 21:53:13,299 - ERROR - No input detected during speech recognition.
2024-12-05 21:53:26,227 - INFO - Initializing Speech-to-Speech Bot...
2024-12-05 21:53:34,161 - ERROR - Speech Recognition could not understand the audio.
2024-12-05 21:53:34,193 - ERROR - No input detected during speech recognition.
2024-12-05 21:53:43,618 - INFO - Initializing Speech-to-Speech Bot...
2024-12-05 21:54:50,615 - INFO - Initializing Speech-to-Speech Bot...
2024-12-05 21:55:08,137 - INFO - Initializing Speech-to-Speech Bot...
2024-12-05 22:55:14,665 - INFO - Initializing Speech-to-Speech Bot...
2024-12-05 22:55:23,605 - ERROR - Speech Recognition could not understand the audio.
2024-12-05 22:55:23,627 - ERROR - No input detected during speech recognition.
2024-12-05 22:55:29,954 - INFO - Initializing Speech-to-Speech Bot...
2024-12-05 22:55:45,216 - ERROR - Speech Recognition could not understand the audio.
2024-12-05 22:55:45,245 - ERROR - No input detected during speech recognition.
2024-12-05 22:56:24,521 - INFO - Initializing Speech-to-Speech Bot...
2024-12-05 22:56:43,319 - INFO - Imported existing <module 'comtypes.gen' from 'C:\\Users\\lodhi\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\comtypes\\gen\\__init__.py'>
2024-12-05 22:56:43,321 - INFO - Using writeable comtypes cache directory: 'C:\Users\lodhi\AppData\Local\Programs\Python\Python312\Lib\site-packages\comtypes\gen'
2024-12-05 22:56:47,128 - INFO - Initializing Speech-to-Speech Bot...
2024-12-05 22:57:24,661 - INFO - Initializing Speech-to-Speech Bot...
2024-12-05 22:57:45,462 - ERROR - Speech Recognition could not understand the audio.
2024-12-05 22:57:45,529 - ERROR - No input detected during speech recognition.
